--- /usr/lib/live/build/binary_grub-efi	2023-02-19 00:09:59.711144321 +0400
+++ /usr/lib/live/build/binary_grub-efi.orig	2023-02-19 00:10:30.827749004 +0400
@@ -255,7 +255,7 @@
 # directories: EFI EFI/boot boot boot/grub
 size=\$((\$size + 4096 * 4))
 
-blocks=\$(((\$size / 1024 + 55) / 32 * 32 ))
+blocks=\$(((\$size / 1024 + 1540) / 32 * 32 ))
 
 rm -f ${_CHROOT_DIR}/grub-efi-temp/boot/grub/efi.img
 # The VOLID must be (truncated to) a 32bit hexadecimal number
